cloud-architecture/custom-apps-and-consolidation/db-reconciliation/files/MF_DP_Recon_Source.sql

```diff
@@ -0,0 +1,236 @@
+--# Copyright (c) 2023, Oracle and/or its affiliates.
+--# Licensed under the Universal Permissive License v1.0 as shown at https://oss.oracle.com/licenses/upl.
+--# Purpose: Script to capture Source Objects counts & Metadata for Reconciliation after Migration/Upgrade Database.
+--# The purpose of this script is to verify the migrated data where the target data is compared against original source data.
+--# @author: Ananda Ghosh Dastidar
+
+clear screen
+set serveroutput on
+set feed off
+set echo off;
+set verify off;
+SET linesize 32767;
+SET LONG 999999;
+SET SHOW OFF;
+SET TRIMS ON;
+SET TERM OFF;
+
+define gv_dp_version=3.0
+col cur_timestamp new_value cur_timestamp
+SET TERMOUT OFF;
+SELECT CURRENT_TIMESTAMP cur_timestamp FROM DUAL;
+SET TERMOUT ON;
+PROMPT
+PROMPT OOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
+PROMPT O                                                 ORACLE CLOUD LIFT SERVICES                                                O
+PROMPT O                                         RECONCILIATION REPORT - SOURCE STEPS v&gv_dp_version                                         O
+PROMPT O                                                 ( Oracle 11g onwards )                                                    O
+PROMPT O                                              &cur_timestamp                                          O
+PROMPT OOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
+PROMPT
+select i.INSTANCE_NAME,i.HOST_NAME,i.VERSION,to_char(i.STARTUP_TIME,'YYYY/MM/DD HH:MI:SS') open_TIME 
+from sys.v_$INSTANCE i;
+PROMPT
+PROMPT
+SET PAGESIZE 0;
+SET HEADING OFF;
+SET VERIFY OFF;
+SET ECHO OFF;
+SET PAGES 0;
+
+ACCEPT user_response PROMPT 'Please press Enter to continue : '
+
+--Source check and validation
+SET TERM OFF;
+SPOOL raise_exit.sql;
+DECLARE
+	v_mode varchar2(20);
+	c INT;
+BEGIN
+	select ltrim(open_mode) into v_mode from v$database;
+	IF (v_mode<>'READ WRITE') THEN
+		dbms_output.put_line('PROMPT Failed...Source Database is in '||v_mode||' Mode.');

```

Comment:
Space after ellipsis like "failed ... Source database is in".

Reply:
This seems to be fine.